frontend/src/views/Market.vue

```diff
@@ -436,13 +494,19 @@ interface Data {
   waitingMarketOutcome: boolean;
   nftPricesById: Record<string, string>;
   characterMarketTax: string;
-  weaponMarketTax: string ;  characterShowLimit: number;
+  weaponMarketTax: string ;
+  characterShowLimit: number;
   weaponShowLimit: number;
   allListingsAmount: number;
   currentPage: number;
   browseTabActive: boolean;
   listingSellPrice: string;
   priceChangeModal: boolean;
+  weaponTransactionHistoryData: any;
```

Comment:
why don't these use IWeaponHistory etc?

Reply:
changed data types to their respective associations